jsonnet/telemeter/client/metrics.json

```diff
@@ -0,0 +1,8 @@
+[
+    "{__name__=\"up\"}",
```

Comment:
It would be valid json if we used single quotes around these? If so maybe a readability addition.

Usually wouldn't be this nit-picky but given that this is probably going to be a point of collaboration with other teams, this is probably something we want to make as easy as possible.

Reply:
yes, this is a good point actually.

Reply:
JSON requires double quotes around all string values :/. Or do you mean single quotes around the label values?

Reply:
Sorry yes, should have been more explicit. Label selectors typically allow using single quotes. But now that you mention it, I actually have no idea about the federation endpoint.

Reply:
I'll run a quick experiment to check :+1: